workshops/models.py

```diff
@@ -1085,10 +1085,15 @@ def __str__(self):
 class Role(models.Model):
     '''Enumerate roles in workshops.'''
 
-    name       = models.CharField(max_length=STR_MED)
+    name = models.CharField(max_length=STR_MED)
+    verbose_name = models.CharField(max_length=STR_LONG,
+                                    null=False, blank=True, default='')
+    show_in_training_request_form = models.BooleanField(
+        default=False,
+        verbose_name='include in training request form?')
```

Comment:
I've just deleted `show_in_training_request_form` role, so no need for hardcoding anything.